packages/oui-field/README.md

```diff
@@ -6,158 +6,183 @@
-            name="os"
-            ng-model="os"
-            class="oui-select__input"
-            required>
-            <option ng-value="undefined">Select the OS</option>
-            <option value="freebsd">FreeBSD</option>
-            <option value="linux">Linux</option>
-            <option value="osx">OSX</option>
-            <option value="windows">Windows</option>
-        </select>
-        <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i>
-    </label>
-</oui-field>
-
-<oui-field label="{{'Recovery OS'}}" size="m">
-    <oui-select name="recovery_os"
-        model="$ctrl.recoveryOs"
-        data-title="Select the recovery OS"
-        placeholder="Select the recovery OS..."
-        items="$ctrl.osList"
-        required
-        match="label"
-        data-align="start">
-        <span ng-bind="$item.label"></span>
-    </oui-select>
-</oui-field>
+<form novalidate name="selectForm">
+    <oui-field label="OS" size="m">
+        <label class="oui-select">
+            <select
+                id="os"
+                name="os"
+                ng-model="os"
+                class="oui-select__input"
+                required>
+                <option ng-value="undefined">Select the OS</option>
+                <option value="freebsd">FreeBSD</option>
+                <option value="linux">Linux</option>
+                <option value="osx">OSX</option>
+                <option value="windows">Windows</option>
+            </select>
+            <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i>
+        </label>
+    </oui-field>
+
+    <oui-field label="Recovery OS" size="m">
+        <oui-select name="recovery_os"
+            model="$ctrl.recoveryOs"
+            data-title="Select the recovery OS"
+            placeholder="Select the recovery OS..."
+            items="$ctrl.osList"
+            required
+            match="label"
+            data-align="start">
+            <span ng-bind="$item.label"></span>
+        </oui-select>
+    </oui-field>
+</form>
 ```
 
 ### Input numeric
 
 ```html:preview
-<oui-field label="{{'Number of servers'}}"
-    help-text="Select the number of servers in your cluster">
-    <oui-numeric
-        id="serversCount"
-        name="serversCount"
-        model="$ctrl.serversCount"
-        min="1">
-    </oui-numeric>
-</oui-field>
+<form novalidate name="numericForm">
+    <oui-field label="Number of servers"
+        help-text="Select the number of servers in your cluster">
+        <oui-numeric
+            id="serversCount"
+            name="serversCount"
+            model="$ctrl.serversCount"
+            min="1">
+        </oui-numeric>
+    </oui-field>
 </form>
 ```
 
+### Input calendar
+
+```html:preview
+<form novalidate name="calendarForm">
+    <oui-field label="Automatic renewal"
+        size="xl">
+        <oui-calendar
+            id="date"
+            name="date"
+            model="$ctrl.dateModel"
+            required>
```

Comment:
Combination with with min-date and max-date could be interesting to test validations.